src/Form/Form.js

```diff
@@ -374,12 +375,17 @@ class Form {
         if (hasIcon) {
             this.addAutofillStyles(input)
             this.addListener(input, 'mousemove', (e) => {
+                if (wasAutofilledByChrome(input)) return
```

Comment:
When Chrome autofills it adds styles that can't be overridden. This causes Dax to become invisible but our listeners were still working (showing the pointer + opening tooltip on click). This is now the expected behavior. 
![CleanShot 2023-03-07 at 13 23 04](https://user-images.githubusercontent.com/1828326/223654189-76f3c722-ebad-4a9f-bc08-2e1979c7440c.gif)